syntax = "proto3";

package suggestions;

service SuggestionsService {
  rpc GetSuggestions (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
  string order_json = 1;
}

message SuggestionsResponse {
  repeated Book suggestions = 1;
}

message Book {
  string id = 1;
  string title = 2;
  string author = 3;
  string description = 4;
  string img = 5;
  int32 price = 6;
}
